packages/hebao_v1/test/testGuardians.ts

```diff
@@ -50,16 +49,14 @@ contract("GuardiansModule", (accounts: string[]) => {
     pendingPeriod = (
       await defaultCtx.guardianModule.pendingPeriod()
     ).toNumber();
-    confirmPeriod = (
-      await defaultCtx.guardianModule.confirmPeriod()
-    ).toNumber();
+
   });
 
   beforeEach(async () => {
     ctx = await createContext(defaultCtx);
   });
 
-  [false, true].forEach(function(metaTx) {
+  [false/*, true*/].forEach(function(metaTx) {
```

Comment:
Seems like meta tx can be enabled here?

Reply:
Sorry, I disabled it for debugging. enabled.